bin/hdfcoinc/pycbc_apply_rerank

```diff
@@ -0,0 +1,95 @@
+#!/bin/env python
+"""Rewrite statmap file with a reranking of the statistic values"""
```

Comment:
The code doesn't actually seem to do this. From this description I imagine a code where you give it a statmap file and a new statistic (e.g. `new_snr`) and the file gets regenerated. This seems to be doing something else, using this followup_file, which I think needs to be described here.